client/core/trade.go

```diff
@@ -2851,14 +2861,25 @@ func (c *Core) sendRedeemAsync(t *trackedTrade, match *matchTracker, coinID, sec
 // mutex lock held for writes.
 func (t *trackedTrade) confirmRedemption(match *matchTracker) {
 	confirmer, isConfirmer := t.wallets.toWallet.Wallet.(asset.RedemptionConfirmer)
-	if !isConfirmer {
+	if !isConfirmer || (match.redemptionConfs > 0 && match.redemptionConfs >= match.redemptionConfsReq) { // already there, stop checking
```

Comment:
This is the only difference from the master version of this fix.  On master, all wallets are confirmers, but not in 0.6, so we merge these cases where handling is the same (mark it confirmed if we have submitted the redeem or it's self-governed).